Core/src/org/sleuthkit/autopsy/geolocation/datamodel/GeolocationManager.java

```diff
@@ -0,0 +1,123 @@
+/*
+ *
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2019 Basis Technology Corp.
+ * contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.geolocation.datamodel;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import org.sleuthkit.datamodel.SleuthkitCase;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ * Static functions for the creations of geolocation points for artifacts.
+ *
+ */
+public final class GeolocationManager {
+
+    /**
+     * Add a private constructor to silence codacy warning about making this
+     * class a utility class as I suspect this class may grow when filtering is
+     * added.
+     */
```

Comment:
This comment was a reminder to ask you about this...  the codacy message seemed to indicate that I either needed to change the name of the class to a utility class or add a private constructor.   

I added the Manager class really to just be consistent with other code I had worked with.  Sometimes I struggle with what I think is right vs. being consistent with existing code.

 Does it make sense to continue to have this class stand alone?  Especially given the timing of your email yesterday?

Reply:
I'm thinking the Codacy warning has nothing to do with the name - it comes up when you have any class with only static methods that does not have a private constructor - such classes should always have a private constructor to prevent instantiation. If I am mistaken about the warning, please show it to me...we don't need Codacy telling us what to name things.

We coul dperhaps test it out with my Codacy plugin, if need be...

Reply:
This is where I got all confused.  Codacy said that because the class had all static functions that I should rename it to Utility\Helper or add a private constructor.  I forget if that is two different messages or one.

Reply:
I could see that if the "or" is actually an "and". Adding the private constructor is the right thing to do. The naming convention of course does not change the fact that classes would all static methods should not be instantiated. I tend to want to think twice before making a Utility/Helper class, since they can become very loosely cohesive. We have the same problem on a different scale with the coreutils package, which incidentally, is very hard to find things in.